packages/TorneloScoresheet/src/style/images.ts

```diff
@@ -32,3 +33,4 @@ export const ICON_HASTAG = ICON_HASHTAG_ASSET;
 export const ICON_REPEAT = ICON_REPEAT_ASSET;
 export const ICON_UNDO = ICON_UNDO_ASSET;
 export const ICON_CLOCK = ICON_CLOCK_ASSET;
+export const ICON_ARBITER_MODE = ICON_ARBITER_MODE_ASSET;
```

Comment:
If we resuse the component i'm talking about, we should import the gears icon instead of the whole button image here, ie import ICON_GEARS instead of ICONS_ARBITER_MODE